Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -653,4 +653,16 @@ final class StringExtensionsTests: XCTestCase {
         str.padEnd(2)
         XCTAssertEqual(str, "str")
     }
+    
```

Comment:
`Locale.current` might change from one device to another, it would be better to use a specific `Locale` here :)